### PR TITLE
KBC-1825 fix data preview with numeric column type

### PR DESCRIPTION
**Pull request by fejjo**
Jira: https://keboola.atlassian.net/browse/KBC-1825
Main PR: https://github.com/keboola/connection/pull/3380

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Backend/Exasol/TableDefinitionOperationsTest.php [hunk 1]
```diff
@@ -53,6 +54,89 @@ private function createTableDefinition()
         return $this->_client->createTableDefinition($bucketId, $data);
     }
 
+    public function testDataPreviewForTableDefinitionWithDecimalType()
+    {
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
+
+        $tableDefinition = [
```
Comment (jirkasemmler) [inline tests/Backend/Exasol/TableDefinitionOperationsTest.php:61]: imho bych trochu rozsiril ty typy co tam zkousime. Dal bych jednoho zastupce z kazdeho BaseTypu a pak vsechny divnotypy co identifikujeme jako string (binary, unique, money...)

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
imho musime dotestovat vice tech typu, viz conenction

File: tests/Backend/Exasol/TableDefinitionOperationsTest.php (hunk 1 above)
Comment (jirkasemmler) [reply tests/Backend/Exasol/TableDefinitionOperationsTest.php:61]: sorry, strcil jsem to k EXA ale melo to jit k Synapse tride. Ale to je ve vysledku jedno

**Comment by fejjo** [comment]
Divno typy pridane diky :)

**Review by jirkasemmler — APPROVED** [review]
lgtm

**Pull request merged**